spec/index.bs

```diff
@@ -1253,10 +1253,22 @@ The default is used if a value is not explicitly specified in {{GPUDeviceDescrip
         The maximum value of the product of the `workgroup_size` dimensions for a
         compute stage {{GPUShaderModule}} entry-point.
 
-    <tr><td><dfn>maxComputeWorkgroupDimensions</dfn>
-        <td>{{GPUExtent3D}} <td>Per-component Higher <td>[256, 256, 64]
+    <tr><td><dfn>maxComputeWorkgroupX</dfn>
```

Comment:
Bit unclear whether this is "max compute workgroups in X" (i.e. max dispatch x), or "max compute workgroup size in X." Maybe `maxComputeWorkgroupWidth` or `maxComputeWorkgroupSizeX`?

Reply:
I chose X/Y/Z rather than Width/Height/Depth because the wgsl [`workgroup_size` definition](https://gpuweb.github.io/gpuweb/wgsl/#attribute-workgroup_size) talks about it in terms of X/Y/Z:

> Specifies the x, y, and z dimensions of the workgroup grid for the compute shader.

That said, I think `maxComputeWorkgroupSizeX`, etc, is a good clarification.